discovery-infra/test_infra/utils/utils.py

```diff
@@ -697,13 +704,20 @@ def get_openshift_version(default=consts.DEFAULT_OPENSHIFT_VERSION):
             )
         return stdout
 
-    return get_env("OPENSHIFT_VERSION", default)
+    version = get_env("OPENSHIFT_VERSION")
+    if version:
+        return version
 
+    if allow_default:
+        return get_default_openshift_version()
 
```

Comment:
at the end of the function, return ``None`` explicitly

Reply:
why? In case no return statement, `None` is returned

Reply:
I know, that's why I said explicit
That's the styling choice that I prefer